src/compiler/types.ts

```diff
@@ -2445,7 +2445,7 @@ namespace ts {
         cachePath: string;                              // The path to the typings cache
         projectRootPath: string;                        // The path to the project root directory
         safeListPath: string;                           // The path used to retrieve the safe list
-        packageNameToTypingLocation: Map<string>;       // The map of package names to their cached typing locations
+        packageNameToTypingLocation: Map<Path>;       // The map of package names to their cached typing locations

```

Comment:
In shims.ts->discoverTypings() we JSON.parse the discoverTypingsJson from the host:
const info = <DiscoverTypingsInfo>JSON.parse(discoverTypingsJson);
where packageNameToTypingLocation is a Map<string> - will this be an issue?